pkg/operator/controllers/muo/muo_controller.go

```diff
@@ -0,0 +1,97 @@
+package muo
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"strings"
+	"time"
+
+	"github.com/sirupsen/logrus"
+	extensionsclient "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/builder"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+
+	arov1alpha1 "github.com/Azure/ARO-RP/pkg/operator/apis/aro.openshift.io/v1alpha1"
+	aroclient "github.com/Azure/ARO-RP/pkg/operator/clientset/versioned"
+	"github.com/Azure/ARO-RP/pkg/operator/controllers"
+	"github.com/Azure/ARO-RP/pkg/util/dynamichelper"
+)
+
+const (
+	CONFIG_NAMESPACE string = "rh.srep.muo"
+	ENABLED          string = CONFIG_NAMESPACE + ".enabled"
+	MANAGED          string = CONFIG_NAMESPACE + ".managed"
+	PULLSPEC         string = CONFIG_NAMESPACE + ".pullspec"
```

Comment:
* First upper case char makes this const exported which seems unnecessary as it is not being used outside of the package
* We use camel case

We need to address this across all controllers (not in this PR, but in general).

Reply:
I ideally wanted to be able to use them in things like the E2E tests.

Also, are global constants not capital snake case, or is that just a C thing?